src/app/admin/components/filter-search/filter-search.component.html

```diff
@@ -0,0 +1,69 @@
+
+  <div class="list">
+    <div #listInner class="list-inner">
+      <div class="list-top">
+        <div class="wrapper">
+          <div class="filter-wrapper">
+            <div class="btn-group to-right">
+                <button class="button neutral filters-button" (click)="toggleCollectionMenu($event)"><span tip="{{'Clear filtering criteria'}}" tipLocation="{{'above'}}" *ngIf="filters.size" (click)="clearFilters(); $event.stopPropagation()"><i class="far fa-times"></i></span> All Collections <i class="fas fa-chevron-down"></i></button>
+                <button class="button neutral filters-button" (click)="toggleFilterMenu($event)"><span tip="{{'Clear filtering criteria'}}" tipLocation="{{'above'}}" *ngIf="filters.size" (click)="clearFilters(); $event.stopPropagation()"><i class="far fa-times"></i></span> All Statuses <i class="fas fa-chevron-down"></i></button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+<clark-context-menu (id)="filterMenu = $event" (close)="toggleFilterMenu(undefined)">
+    <div #contextMenu class="filter-menu">
+      <ul>
+        <li class="unreleased" (click)="toggleStatusFilter('unreleased')">
+          <div *ngIf="filters.get('unreleased')" class="indicator"></div>
+          <i class="far fa-eye-slash"></i>Unreleased
+        </li>
+        <li class="waiting" (click)="toggleStatusFilter('waiting')">
+          <div *ngIf="filters.get('waiting')" class="indicator"></div>
+          <i class="far fa-hourglass"></i>Waiting for Review
+        </li>
+        <li class="review" (click)="toggleStatusFilter('review')">
+          <div *ngIf="filters.get('review')" class="indicator"></div>
+          <i class="far fa-sync"></i>Under Review
+        </li>
+        <li class="released" (click)="toggleStatusFilter('released')">
+          <div *ngIf="filters.get('released')" class="indicator"></div>
+          <i class="far fa-eye"></i>Released
+        </li>
+        <li class="denied" (click)="toggleStatusFilter('rejected')">
+          <div *ngIf="filters.get('rejected')" class="indicator"></div>
+          <i class="far fa-ban"></i>Denied
```

Comment:
Instead of hardcoding each list item, try using an ngFor.

Reply:
@nrwinner what are your thoughts on this? It will require the component class to have the icon class names built in. I see no architectural issue with it since it is still the component. Thoughts?

Reply:
@DonSeannelly @nvisal1 I'm on board with the introduction of ngFor here. I agree that we don't break any architectural by giving the component class names for icons.